gen3/bin/s3.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Describe and create s3 buckets
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+help() {
+  gen3 help s3
+}
+
+gen3_s3() {
+  command="$1"
+  shift
+  case "$command" in
+    'list'|'ls')
+      list "$@"
+      ;;
+    'new')
+      new "$@"
+      ;;
+    'get')
+      get "$@"
+      ;;
+    *)
+      help
+      ;;
+  esac
+}
+
+list() {
+  if [[ -z "$1" ]]; then
+    gen3_aws_run aws s3 ls
+  else
+    gen3_aws_run aws s3 ls s3://$1
+  fi
+}
+
+new() {
+  local bucketName=$1
+  # do simple validation of bucket name
+  local regexp="^[a-z][a-z0-9\-]*$"
+  if [[ ! $bucketName =~ $regexp ]];then
+    cat << EOF
+ERROR: Bucket name does not meet the following requirements:
+  - starts with a-z
+  - contains only a-z, 0-9, and dashes, "-"
+EOF
+    exit 1
+  fi
+  # if bucket already exists do nothing and exit
+  if [[ -z "$(gen3_aws_run aws s3api head-bucket --bucket $bucketName 2>&1)" ]]; then
+    echo "INFO: Bucket already exists"
```

Comment:
use `gen3_log_err` instead of `echo` - it takes care of piping to `stderr`, and also colors thing red